apps/web/src/state/app/splash.ts

```diff
@@ -11,7 +11,12 @@ export const bootstrap = async (): Promise<BootstrapResult> => {
 export const launchApp = async ({
 	profile,
 }: BootstrapResult): Promise<void> => {
-	if (profile?.email) {
+	const url = window.location.hash;
+	const path = url.slice(1);
+
+	if (path) {
+		await router.navigate(path);
+	} else if (profile?.email) {
 		await router.navigate('/');
 	} else {
 		await router.navigate('/invitation');
```

Comment:
double check this versus `dev` branch, it already updated different from this piece